helm/charts/nats/values.yaml

```
@@ -564,7 +564,7 @@ natsBox:
   container:
     image:
       repository: natsio/nats-box
-      tag: 0.14.3
+      tag: 0.14.4
```

oh dependabot...  why do you match images that are not the image that changed...

```suggestion
      tag: 0.14.3
```